mods/d2k/maps/atreides-02a/atreides02a.lua

```diff
@@ -0,0 +1,111 @@
+
+HarkonnenBase = { HConyard, HPower1, HPower2, HBarracks }
+
+HarkonnenReinforcements = { }
+HarkonnenReinforcements["Easy"] =
+{
+	{ "light_inf", "trike" },
+	{ "light_inf", "trike" },
+	{ "light_inf", "light_inf", "light_inf", "trike", "trike" }
+}
+
+HarkonnenReinforcements["Normal"] =
+{
+	{ "light_inf", "trike" },
+	{ "light_inf", "trike" },
+	{ "light_inf", "light_inf", "light_inf", "trike", "trike" },
+	{ "light_inf", "light_inf" },
+	{ "light_inf", "light_inf", "light_inf" },
+	{ "light_inf", "trike" },
+}
+
+HarkonnenReinforcements["Hard"] =
+{
+	{ "trike", "trike" },
+	{ "light_inf", "trike" },
+	{ "light_inf", "trike" },
+	{ "light_inf", "light_inf", "light_inf", "trike", "trike" },
+	{ "light_inf", "light_inf" },
+	{ "trike", "trike" },
+	{ "light_inf", "light_inf", "light_inf" },
+	{ "light_inf", "trike" },
+	{ "trike", "trike" }
+}
+
+HarkonnenAttackDelay = { }
+HarkonnenAttackDelay["Easy"] = DateTime.Minutes(5)
+HarkonnenAttackDelay["Normal"] = DateTime.Minutes(2) + DateTime.Seconds(40)
+HarkonnenAttackDelay["Hard"] = DateTime.Minutes(1) + DateTime.Seconds(20)
```

Comment:
Is DateTime.Minutes(1) + DateTime.Seconds(20) your prefered over DateTime.Seconds(80) or is it stylistically better? I want to know so we can follow same guidelines across the files.